src/mmw/js/src/draw/models.js

```diff
@@ -6,21 +6,39 @@ var Backbone = require('../../shim/backbone'),
 
 var ToolbarModel = Backbone.Model.extend({
     defaults: {
-        toolsEnabled: true,
```

Comment:
I *think* we may need this or something similar back, as it prevented the UI from kicking off more than one analyze op simultaneously.